test/controllers/ConfirmationController.test.ts

```diff
@@ -0,0 +1,32 @@
+import 'reflect-metadata'
+import {createApplication} from '../ApplicationFactory';
+import * as request from 'supertest'
+import {createSubstituteOf} from '../SubstituteFactory';
+import {expect} from 'chai';
+
+import '../../src/controllers/ConfirmationController';
+import {CONFIRMATION_PAGE_URI} from '../../src/utils/Paths';
+import {OK} from 'http-status-codes';
```

Comment:
I noticed in other controllers we usually put spaces between the curly braces and the name of what is being imported, for example 
`{ expect }` rather than `{expect}`